src/Microsoft.Extensions.OData.Migration/ExtensionMethods.cs

```diff
@@ -0,0 +1,143 @@
+﻿using Microsoft.OData.Edm;

```

Comment:
Per emailed comments, please add copyright and move using statements into namespace.